include/invert_quda.h

```diff
@@ -771,21 +783,29 @@ namespace quda {
     const DiracMatrix &matSloppy;
     bool init;
 
-    Complex *W; // inner product matrix
-    Complex *C; // inner product matrix
-    Complex *alpha;
-    Complex *beta;
-    Complex *phi;
+    bool lambda_init;
+    QudaCABasis basis;
+
+    Complex *Q_AQandg; // Fused inner product matrix
+    Complex *Q_AS; // inner product matrix
+    Complex *alpha; // QAQ^{-1} g
+    Complex *beta; // QAQ^{-1} QpolyS
```

Comment:
clang-format suggested the following change:


```
-    Complex *Q_AQandg; // Fused inner product matrix
-    Complex *Q_AS; // inner product matrix
-    Complex *alpha; // QAQ^{-1} g
-    Complex *beta; // QAQ^{-1} QpolyS
+    Complex* Q_AQandg; // Fused inner product matrix
+    Complex* Q_AS; // inner product matrix
+    Complex* alpha; // QAQ^{-1} g
+    Complex* beta; // QAQ^{-1} QpolyS
```

<!-- identifier: clang-format -->
<!-- File: include/invert_quda.h -->